docs/wait.md

```diff
@@ -120,6 +123,8 @@ function sayThree() print("Three") end
 ###stop(...)
 [<span class="ret boo"></span>](types.md)&nbsp;Stops a currently running Wait function. The only way to obtain these ID numbers is to get them from the return value of a Wait function.
 
+> The returned value is true if a Wait function with this ID was stopped. The returned value is false if a Wait function with this ID either doesn't exist, or has already been stopped.
```

Comment:
I'd probably suggest replacing "returned value" with "return value" (two occurrences).

Otherwise, "value returned _will be_" (i.e. flipped words and changed tense) would work too.